### ISSUE TITLE
long start time

### ISSUE THREAD

**Issue by gdevos010** [martinwhl/Informer-PyTorch-Lightning#1]
Im very interested in this PyTorch lightning version since it has many additional capabilities. However, the runtime seems to be much longer than expected, especially before the epochs start. 

Have you seen this behavior, and do you have any thoughts about why this is happening?

**Comment by martinwhl** [comment]
Sorry for the late reply, and thanks for your attention to this repo. Yes, the waiting time is long before the first epoch starts, and the reason lies in this line of code:
https://github.com/martinwhl/Informer-PyTorch-Lightning/blob/35fc041b58c6693f12723b42e3e381c137a2209e/tasks/forecast.py#L27
By calling `self.save_hyperparameters()`, the arguments of the `__init__()` function of a `LightningModule` is saved to a `yaml` file (`lightning_logs/version_*/hparams.yaml`). As the Informer model is also an argument of `__init__()` in our case, PyTorch Lightning will also save the weights of the model to the hyperparameters configuration file, which takes a long time. If you do not need to re-use the trained model, you can change this line of code to
```Python
    self.save_hyperparameters(ignore=['model'])
```
Hope this answers your question.

**Comment by gdevos010** [comment]
Thank you so much! Im still a bit new to PyTorch lightning

**Comment by martinwhl** [comment]
Update: this issue seems to be solved in Lightning 2.0.

```Python
    self.save_hyperparameters(ignore=['model'])
```
